src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -81,10 +81,10 @@
           </th>
         {% endif %}
         {% if not orderForViewing.delivered %}
-          <th class="text-right product_actions">
+          <th class="text-right product_actions d-print-none">
             <p>{{ 'Actions'|trans({}, 'Admin.Global') }}</p>
           </th>
-        {% endif %}
+          {% endif %}
```

Comment:
```suggestion
        {% endif %}
```

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/880b0ffdaca18f032d2cb941f37e0751b4745479